### PR TITLE
Merge the missing 50

### PR DESCRIPTION
**Pull request by stvstnfrd**
When merging the Django 1.8 code, we, unsurprisingly, pulled in the
`django-1.8-release` tag. As it turned out, this branch did not share a
common ancestor with `edx/master` and was missing a few commits.

It looks as if:
- a feature branch was created
- a tag was created on this feature branch
- the feature branch was rebased on a new master
- the feature branch merged to master

TODO: edX should delete this tag.

missing 50

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@kluo @caesar2164

**Comment by kluo** [comment]
+1

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+- Role: edxapp
+  - Removed XmlModuleStore from the default list of modulestores for the LMS.
+  - EDXAPP_XML_MAPPINGS variable no longer exists by default and is not used by the edxapp role.
```
Comment (stvstnfrd) [inline CHANGELOG.md]: I checked; we do not have this in secrets.

**Comment by caesar2164** [comment]
cool, the conflicts are banal.

👍

**Pull request merged**